.github/workflows/ci_ut_ext_blob_workflow.yml

```diff
@@ -52,13 +52,30 @@ jobs:
         name: codecov
         fail_ci_if_error: false
 
-    - name: Notify dedicated teams channel
-      uses: jdcargile/ms-teams-notification@v1.4
-      if: failure()
-      with:
-        github-token: ${{ github.token }} # this will use the runner's token.
-        ms-teams-webhook-uri: ${{ secrets.MS_TEAMS_WEBHOOK_URI }}
-        notification-summary: "Python Extension Blob UT CI Failed for Python ${{ matrix.python-version }}"
-        notification-color: 17a2b8
-        timezone: America/Denver
-        verbose-logging: false
+
+  send-notification:
+    runs-on: ubuntu-latest
+    needs: build
+    if: always()
+    steps:
+      - name: Notify dedicated teams channel on failure
+        if: ${{ always() && needs.build.result == 'failure' }}
+        uses: jdcargile/ms-teams-notification@v1.4
+        with:
+          github-token: ${{ github.token }}
+          ms-teams-webhook-uri: ${{ secrets.MS_TEAMS_WEBHOOK_URI }}
+          notification-summary: "Python Extension Blob UT CI Failed"
```

Comment:
nit: rename to "Python Extension Blob CI Failed" since we don't have separate unit / E2E tests

Reply:
updated